src/runtime_src/core/pcie/driver/linux/xocl/userpf/xocl_drm.c

```diff
@@ -630,6 +645,7 @@ void xocl_drm_fini(struct xocl_drm *drm_p)
 	xocl_drvinst_release(drm_p, &hdl);
 
 	xocl_cleanup_mem(drm_p);
+	xocl_cleanup_memory_manager(drm_p);
```

Comment:
In xocl_drm_init(), you called xocl_init_drm_memory_manager(). I will expect that xocl_cleanup_drm_memory_manager() should be called in xocl_drm_fini(). Of course, we can call xocl_cleanup_memory_manager() as well.

Reply:
Modified the code for better readability. Please check.